### PR TITLE
Fix BootstrapOSImage link in install-config.yaml file

### LINKED ISSUE #424: Document bootstrapOSImage
https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#ipi-install-creating-an%20rhcos-images-cache_ipi-install-prerequisites describes how to set up an image cache for the RHCOS images, but does not document the `bootstrapOSImage` in `install-config.yaml`.

Without that setting, the image cache will not be used.

### PR DESCRIPTION
**Pull request by DirectedSoul1**
# Description

Adding the `bootstrapOSImage` and `ClusterOSImage` in `install-config.yaml` file 

Fixes #424 

## Type of change

Please select the appropiate options:

- [x] This change is a documentation update

### CODE REVIEW

File: ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  register: ingress_vip
+  tags: vip_status
+
+- name: Fail if Ingress VIP already exists
+  fail:
+    msg: |
+      The Ingress VIP is being active in your cluster
+      please ensure to turn off all the nodes before deployment
+  when: ingress_vip.status == 503
+  tags: vip_status
```
Comment (rlopez133) [inline ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml:30]: This file is unrelated to your bootstrap changes.

File: ansible-ipi-install/roles/installer/tasks/main.yml [hunk 2]
```diff
@@ -43,6 +43,8 @@
   tags: extramanifests
 - include_tasks: 55_customize_filesystem.yml
   tags: customfs
+- include_tasks: 57_check_vip_status.yml
+  tags: vip_status
 - include_tasks: 59_cleanup_bootstrap.yml
   tags: cleanup
 - include_tasks: 60_deploy_ocp.yml
```
Comment (rlopez133) [inline ansible-ipi-install/roles/installer/tasks/main.yml:50]: unrelated to this PR

File: documentation/ipi-install/modules/ipi-install-config-yaml.adoc [hunk 3]
```diff
@@ -24,6 +24,8 @@ platform:
     dnsVIP: <dns-ip>
     provisioningNetworkInterface: <NIC1>
     provisioningNetworkCIDR: <CIDR> # i.e. fd00:1101::1/64
+    bootstrapOSImage: http://<Disconnected-Registry-fqdn>/rhcos-qemu-sha256
+    clusterOSImage: http://<Disconnected-Registry-fqdn>/rhcos-openstack-sha256 
     provisioningBridge: provisioning
     hosts:
       - name: openshift-master-0
```
Comment (rlopez133) [inline documentation/ipi-install/modules/ipi-install-config-yaml.adoc:31]: The fix isn't this simple. These pieces are optional. So they wouldn't not be part of the install-config.yaml directly. They would be part of the https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#additional-install-config-parameters_ipi-install-prerequisites , but in actuality they would be part of this PR which isn't merged yet: https://github.com/openshift-kni/baremetal-deploy/pull/418

File: ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml (hunk 1 above)
Comment (DirectedSoul1) [reply ansible-ipi-install/roles/installer/tasks/57_check_vip_status.yml]: I accidentally added the old file changes to this PR , so removing the same, Thanks :)

File: ansible-ipi-install/roles/installer/tasks/main.yml (hunk 2 above)
Comment (DirectedSoul1) [reply ansible-ipi-install/roles/installer/tasks/main.yml]: removing this file as this is not related to the #424 issue.
